src/providers/google/ai.studio.ts

```diff
@@ -195,6 +195,9 @@ export class AIStudioChatProvider extends AIStudioGenericProvider {
           maxOutputTokens: this.config.maxOutputTokens,
         }),
         ...this.config.generationConfig,
+        ...(this.config.generationConfig?.thinkingConfig && {
```

Comment:
Won't `thinkingConfig` be inherited by default in `...this.config.generationConfig` on 197?